setup/functions

```diff
@@ -318,7 +318,7 @@ EOT
 
 function doForAllModules {
     # Install required node packages
-    for idx in client shared server test-embedd ivis-core/client ivis-core/server ivis-core/shared ivis-core/embedding; do
+    for idx in client shared server test-embedd ivis-core/client ivis-core/server ivis-core/shared ivis-core/embedding ./; do
```

Comment:
As stated in the comment to the new `package.json` file, I don't think it should be installed automatically with the project so I would keep the original line here.

Also, it doesn't work properly when `ivis-core` is installed inside `fivis` or other extension.